crates/biome_module_graph/src/js_module_info.rs

```diff
@@ -141,10 +139,10 @@ pub struct JsModuleInfoInner {
     /// The keys are the names of the exports, where "default" is used for the
     /// default export. See [JsExport] for information tracked per export.
     ///
-    /// Re-exports are tracked in this map as well. The exception are "blanket"
+    /// Re-exports are tracked in this map as well. The exception is "blanket"
     /// re-exports, such as `export * from "other-module"`. Those are tracked in
     /// [Self::forwarding_exports] instead.
-    pub exports: BTreeMap<Text, JsExport>,
+    pub exports: Exports,
```

Comment:
Why do this for `Exports`, but not the imports or blanket reexports? It seems nothing special is added to the `Debug` implementation?